update/ansible/playbook/tasks/files/datasources.yml

```diff
@@ -5,7 +5,7 @@ datasources:
   orgId: 1
   type: prometheus
   access: proxy
-  url: http://127.0.0.1:9090/prometheus/
+  url: http://127.0.0.1:1280/prometheus/
```

Comment:
does VM gateway use the same port?

Reply:
No, they use different.

Reply:
Sure, that's configurable. Any specific reason for this change?

Reply:
Changed to `8430`.
`8431` is used by vmgateway.